packages/documentation/copy/en/handbook-v2/Classes.md

```diff
@@ -1269,6 +1269,32 @@ const m = new someClass("Hello, world");
 //    ^?
 ```
 
+## Class Constructors
+
+Calling a constructor of a class returns an instance, which can be described using the `InstanceType` type.

```

Comment:
I'm not quite sure what the right wording is, but I don't think that "calling" is the right one as that can imply `Point(1, 2)` works (not `new Point(1, 2)`), even though those are two things. Potentially our docs already describe it as `new`-ing? Or as just instantiation?

Potentially what this section should be is a description of construct signatures? Or, just a link to https://www.typescriptlang.org/docs/handbook/2/functions.html#construct-signatures 

Of course, it's unfortunate that classes and functions are described separately but have this intertwining behavior.

Reply:
how about I change it to "Instantiating a class returns a class instance, ..."?

Reply:
Hey @jakebailey -- following up on this question please

Reply:
Personally I think "Instantiating a class" is the right call probably, there isn't really language like this in [the spec](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-new-operator) nor [in MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new) with that term, but it's such a globally used term in many languages that I think it fits. 

Calling it something like a newable  function is a bit more accurate but I think moves too far into 'abstract but correct' terminology

Reply:
If anything I'd go for "instantiating a class produces a class instance", or "The `new` operator instantiates a class ...", etc.

So like:

> Classes are instantiated with `new`. Given the type of a class itself, the `InstanceType` utility type will model this operation.

But, I'm a little iffy on this whole PR just from the point of view that it's duplicating content by describing both `new` (whose docs I linked to), but then also `InstanceType`, which is described on the utility type page. If anything, I think providing links to each section may be most helpful.

Reply:
I'm okay with dropping it if y'all don't want it. Honestly at this point I'm a little exhausted with all the back and forth.

Reply:
Sorry about that.

If you want something concrete to finish this, then I would probably just use a variation of my text then use `[]()` links to point the word "constructor" to `/docs/handbook/2/functions.html#construct-signatures` and `` `InstanceType` `` to `/docs/handbook/utility-types.html#instancetypetype`.

You could even just skip the code sample and let those sections speak for themselves.

Reply:
Okay! How about now? I accepted your suggestion and tweaked a little to better match those links you shared